src/wsrep_provider_v26.cpp

```diff
@@ -421,14 +422,35 @@ namespace
         }
     }
 
-    int encrypt_cb(void*                 /* app ctx */,
-                                 wsrep_enc_ctx_t*      /*ctx*/,
-                                 const wsrep_buf_t*    /*input*/,
-                                 void*                 /*output*/,
-                                 wsrep_enc_direction_t /*direction*/,
-                                 bool                  /*final*/)
+    int encrypt_cb(void*                 app_ctx,
+                   wsrep_enc_ctx_t*      enc_ctx,
+                   const wsrep_buf_t*    input,
+                   void*                 output,
+                   wsrep_enc_direction_t direction,
+                   bool                  last)
     {
-        return 0;
+        assert(app_ctx);
+        wsrep::server_state& server_state(
+            *static_cast<wsrep::server_state*>(app_ctx));
+
```

Comment:
```suggestion
assert(server_state.encryption_service());
if (server_state.encryption_service() == 0)
{
   wsrep::log_error() << "Encryption service not defined in encrypt_cb()";
   return -1;
}
```

Reply:
Encryption_cb is already set only if there is service provided & enabled. But this will not hurt to add for extra protection.